Mathlib/Tactic/GCongr/CoreAttrs.lean

```diff
@@ -0,0 +1,13 @@
+/-
+Copyright (c) 2024 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Tactic.GCongr.Core
+
+/-!
+# gcongr attributes for core lemmas
```

Comment:
```suggestion
# gcongr attributes for lemmas outside Mathlib
```

Reply:
I don't know if Batteries lemmas should go there (whenever we'll decide to add `attribute [gcongr]` to some of these lemmas), because this would mean adding imports to this file. I'll update the docstring.